test/e2e/storage/csi_volumes.go

```diff
@@ -223,3 +227,144 @@ func (g *gcePDCSIDriver) cleanupCSIDriver() {
+			csiDriverSecretAccessClusterRoleName,
+		},
+
+		f:      f,
+		config: config,
+	}
+}
+
+func (r *rbdCSIDriver) createStorageClassTest(node v1.Node) storageClassTest {
+	return storageClassTest{
+		name:        "csi-rbdplugin",
+		provisioner: "csi-rbdplugin",
+		parameters: map[string]string{
+			"monitors": r.serverIP,
+			"pool":     "rbd",
+			"csiProvisionerSecretName":      r.secret.Name,
+			"csiProvisionerSecretNamespace": r.config.Namespace,
+		},
+		claimSize:          "1Gi",
+		expectedSize:       "1Gi",
+		nodeName:           node.Name,
+		skipWriteReadCheck: true,
+	}
+}
+
+func (r *rbdCSIDriver) createCSIDriver() {
+	By("deploying csi rbd driver")
+	f := r.f
+	cs := f.ClientSet
+	config := r.config
+	r.controllerServiceAccount = csiServiceAccount(cs, config, "rbd-controller", false /* setup */)
+	r.nodeServiceAccount = csiServiceAccount(cs, config, "rbd-node", false /* setup */)
+	csiClusterRoleBindings(cs, config, false /* setup */, r.controllerServiceAccount, r.controllerClusterRoles)
+	csiClusterRoleBindings(cs, config, false /* setup */, r.nodeServiceAccount, r.nodeClusterRoles)
+	deployRbdCSIDriver(cs, config, false /* setup */, f, r.nodeServiceAccount, r.controllerServiceAccount, r)
+}
+
+func (r *rbdCSIDriver) cleanupCSIDriver() {
+	By("uninstalling csi rbd driver")
+	f := r.f
+	cs := f.ClientSet
+	config := r.config
+	deployRbdCSIDriver(cs, config, true /* teardown */, f, r.nodeServiceAccount, r.controllerServiceAccount, r)
+	csiClusterRoleBindings(cs, config, true /* teardown */, r.controllerServiceAccount, r.controllerClusterRoles)
+	csiClusterRoleBindings(cs, config, true /* teardown */, r.nodeServiceAccount, r.nodeClusterRoles)
+	csiServiceAccount(cs, config, "rbd-controller", true /* teardown */)
+	csiServiceAccount(cs, config, "rbd-node", true /* teardown */)
+}
+
+type cephfsCSIDriver struct {
+	controllerClusterRoles   []string
+	nodeClusterRoles         []string
+	controllerServiceAccount *v1.ServiceAccount
+	nodeServiceAccount       *v1.ServiceAccount
+	serverPod                *v1.Pod
+	secret                   *v1.Secret
+	serverIP                 string
+
+	f      *framework.Framework
+	config framework.VolumeTestConfig
+}
+
+func initCSIcephfs(f *framework.Framework, config framework.VolumeTestConfig) csiTestDriver {
+	return &cephfsCSIDriver{
+		nodeClusterRoles: []string{
+			csiDriverRegistrarClusterRoleName,
+		},
+		controllerClusterRoles: []string{
+			csiExternalAttacherClusterRoleName,
+			csiExternalProvisionerClusterRoleName,
+			csiDriverSecretAccessClusterRoleName,
+		},
+
+		f:      f,
+		config: config,
+	}
+}
+
+func (c *cephfsCSIDriver) createStorageClassTest(node v1.Node) storageClassTest {
+	return storageClassTest{
+		name:        "csi-cephfsplugin",
+		provisioner: "csi-cephfsplugin",
+		parameters: map[string]string{
+			"monitors":        c.serverIP,
+			"provisionVolume": "true",
+			"pool":            "cephfs_data",
+			"csiProvisionerSecretName":      c.secret.Name,
```

Comment:
This changed in with 0.3.0 (along with refurbished docs), please have a look at https://github.com/ceph/ceph-csi/blob/master/docs/deploy-cephfs.md#configuration

```
...
"csiProvisionerSecretName": c.secret.Name,
"csiProvisionerSecretNamespace": c.config.Namespace,
"csiNodeStageSecretName": c.secret.Name,
"csiNodeStageSecretNamespace": c.config.Namespace,
...
```